auctions/README.md

```diff
@@ -0,0 +1,117 @@
+# Auctions
+
+## Auction object
+
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <example: neelam>,
+  'initial_price': <example: 100 dinero>,
+  'bidders_and_bids': [{
```

Comment:
We should separate the API calls
Make a metadata call for auction which contains the number of bidders
And a data call for auction which returns the bidders and their bids

Reduces db call, and fetched objects are smalled